src/config/ConfigService.js

```diff
@@ -18,23 +18,68 @@ export default class ConfigService extends Service {
 
 	initialize() {
 		// Initialization of dynamic configuration
-		const headerLogoFull = document.getElementsByName('header-logo-full')[0]?.content;
-		const headerLogoMini = document.getElementsByName('header-logo-minimized')[0]?.content;
+		const headerLogoFullLight = document.getElementsByName('header-logo-full-light')[0]?.content;
```

Comment:
@petrKavulok Not sure, but I thought we were speaking about that the light logo will not have the `light` appendix

```
const headerLogoFull = document.getElementsByName('header-logo-full')[0]?.content;
```

Its because of backwards compatibility...